From 028b83b9d97582387635d19ad172e9e2e3ed712e Mon Sep 17 00:00:00 2001
From: Reto Zingg <reto.zingg@jollamobile.com>
Date: Tue, 2 Oct 2012 14:24:30 +0300
Subject: [PATCH] include QDebug to remotepluginprocess

Build is failing otherwise with:
error: invalid use of incomplete type 'struct QDebug'
---
 src/remotepluginprocess/debug.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/remotepluginprocess/debug.h b/src/remotepluginprocess/debug.h
index 3b22e01..5328b01 100644
--- a/src/remotepluginprocess/debug.h
+++ b/src/remotepluginprocess/debug.h
@@ -23,6 +23,8 @@
 #ifndef REMOTEPLUGINPROCESS_DEBUG_H
 #define REMOTEPLUGINPROCESS_DEBUG_H
 
+#include <QDebug>
+
 #ifdef DEBUG_ENABLED
     extern int debugLevel;
 
-- 
1.7.12.3

